flake.nix

```diff
@@ -0,0 +1,8 @@
+{
+  inputs.nixpkgs.url = "nixpkgs/nixos-20.03-small";
+
+  outputs = { self, nixpkgs }: {
+    nixosModule = self.nixosModules.impermanence;
+    nixosModules.impermanence = import ./nixos.nix;
```

Comment:
Are both necessary? The wiki entry makes it sound like `nixosModules` is only needed if multiple modules are provided.